### ISSUE TITLE
AngleSharp skips empty "style" attribute

### ISSUE THREAD

**Issue by meziantou** [AngleSharp/AngleSharp#507]
AngleSharp discards empty `style` attributes, so `HasAttribute("style")` returns `false` whereas Google Chrome returns `true` (not tested on other browsers).

```
var config = Configuration.Default.WithDefaultLoader();
var browsingContext = BrowsingContext.New(config);
var content = "<div id='sample' style='' abc=''></div>";

var htmlDocument = await browsingContext.OpenAsync(action => action.Content(content));
var element = htmlDocument.GetElementById("sample");

Console.WriteLine(element.HasAttribute("style")); // false => Should be true
Console.WriteLine(element.HasAttribute("abc")); // true
```

**Comment by FlorianRappl** [comment]
This is actually by-design (I think it was in spec. + browsers behaved like that), but looking at IE, FF, and Chrome it is indeed true that no specialization on the `style` attribute may be needed.

Will need to double check first.

**Comment by FlorianRappl** [comment]
Fixed and will be available in the next release, see PR #508.
